packages/relay/src/lib/services/transactionService/transactionService.ts

```diff
@@ -113,21 +106,16 @@ export default class TransactionService {
 
         const transactionReceipt = transactionRecord.receipt;
 
-        // retrieve operaotr's balance after the execution
-        const operatorBalanceAfter = await this.sdkClient.getBalanceInTinyBars(
-          operatorAccountId,
-          callerName,
-          requestId,
-          200,
+        // calculate transactionRecord fee
+        const hbarToTinybar = Hbar.from(1, HbarUnit.Hbar).toTinybars().toNumber();
+        const exchangeRateInCents = transactionReceipt.exchangeRate!.exchangeRateInCents;
+        txRecordChargeAmount = Math.round(
+          (Constants.TX_RECORD_QUERY_COST_IN_CENTS / exchangeRateInCents) * hbarToTinybar,
```

Comment:
Given that it's already a method with 89 lines, would it not make sense to break out the calculation into its own function?  I could be more readable.
```
function calculateTxRecordChargeAmount(exchangeRate: ExchangeRate): number {
    const exchangeRateInCents = exchangeRate.exchangeRateInCents;
    return Math.round((TX_RECORD_QUERY_COST_IN_CENTS / exchangeRateInCents) * HBAR_TO_TINYBAR);
}
```
```
// Usage
const txRecordChargeAmount = **calculateTxRecordChargeAmount(transactionReceipt.exchangeRate!);**
```

Reply:
Great point! Updated with a new refactored function.